docs/concepts/resource-limits-and-performance-optimization.md

```diff
@@ -36,22 +42,23 @@ If any content or task pane add-in exceeds the preceding thresholds on CPU core
 
-> [!NOTE]
-> Even though only Outlook on Windows (classic) and on Mac monitor resource usage, if either of these clients makes an Outlook add-in unavailable, that add-in also becomes unavailable in Outlook on the web, on mobile devices, and in [new Outlook on Windows](https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627).
+- **Regular expressions response time** - Applies to classic Outlook on Windows only. A default threshold of 1,000 milliseconds for Outlook to evaluate all regular expressions in the manifest of an Outlook add-in. Exceeding the threshold causes Outlook to retry evaluation at a later time.
 
-In addition to the CPU core, memory, and reliability rules, Outlook add-ins should observe the following rules on activation.
+  Administrators can adjust this default threshold value of 1,000 milliseconds by using a group policy or application-specific setting for the `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\Outlook\ActivationAlertThreshold` DWORD value in the Windows registry.
 
-- **Regular expressions response time** - A default threshold of 1,000 milliseconds for Outlook to evaluate all regular expressions in the manifest of an Outlook add-in. Exceeding the threshold causes Outlook to retry evaluation at a later time.
+- **Regular expressions re-evaluation** - A default limit of three times for Outlook to reevaluate all the regular expressions in a manifest. If evaluation fails to evaluate within the time limit three times, Outlook makes the add-in unavailable.
 
-    Administrators can adjust this default threshold value of 1,000 milliseconds by using a group policy or application-specific setting for the **OutlookActivationAlertThreshold** key in the Windows registry.
+  Administrators can adjust this number of times to retry evaluation by using a group policy or application-specific setting. The location of the setting depends on the platform.
 
-- **Regular expressions re-evaluation** - A default limit of 3 times for Outlook to reevaluate all the regular expressions in a manifest. If evaluation fails to evaluate within the time limit 3 times, Outlook makes the add-in unavailable.
+  - **Windows**: The `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\Outlook\ActivationRetryLimit` DWORD value in the Windows registry.
+  - **Mac**: The `ActivationRetryLimit` property list in `~/Library/Preferences`.
 
-    administrators can adjust this number of times to retry evaluation by using a group policy or application-specific setting for the  **OutlookActivationManagerRetryLimit** key in the Windows registry.
+> [!NOTE]
+> Although the regular expressions response time thresholds only apply to Outlook on Windows (classic) and on Mac (classic), if an Outlook add-in becomes unavailable on these clients, the add-in also becomes unavailable for the mailbox on other supported Outlook clients.
```

Comment:
This sounds like we disable the add-in, which we don't do anymore.

Reply:
Thanks for clarifying. I'll remove this.